libs/sdk-ui-ext/src/internal/components/configurationControls/comparison/calculation/CalculationListItemInfoSection.tsx

```diff
@@ -0,0 +1,27 @@
+// (C) 2023 GoodData Corporation
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+type SectionType = "example" | "useIn" | "formula";
+
+interface ICalculationItemInfoProps {
+    info: string;
+    section: SectionType;
+}
+
+const CalculationListItemInfo: React.FC<ICalculationItemInfoProps> = ({ info, section }) => {
+    return (
+        <>
+            <FormattedMessage
+                id={`properties.comparison.calculationType.info.${section}.title`}
+                tagName="h4"
+            />
+            <FormattedMessage
+                id={`properties.comparison.calculationType.as.${info}.info.${section}`}
```

Comment:
Can you please redesign the component so the one string is used as ID instead of template string? For example create a function in this file that would contain `switch` statement that will pick string id based on info or section. We don't like to do it this way because it is hard to find out where the localization string is used or employ automatic tools that checks for unused localization texts. Please fix both messages and possible other places I missed. Thank you

It's mentioned here: https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/localisation.md#4-using-formatmessage-intlformatmessage-and-placeholders
("Do not create message id dynamically.", "defineMessages").

Reply:
This component has been redesigned as per your suggestion!
Thank you for your suggestion!